src/modules/ChannelSettings/components/ModerationPanel/index.tsx

```diff
@@ -24,13 +24,13 @@ import MutedMemberList from './MutedMemberList';
 
 import { useChannelSettingsContext } from '../../context/ChannelSettingsProvider';
 
-const kFormatter = (num: number): string|number => {
+const kFormatter = (num: number): string | number => {
   return Math.abs(num) > 999
     ? `${(Math.abs(num) / 1000).toFixed(1)}K`
     : num;
 };
 
-export default function AdminPannel(): ReactElement {
```

Comment:
We import this component as ModerationPanel everywhere, so good to keep the component name and file name